src/Meshes/meshBuilder.ts

```diff
@@ -546,4 +547,36 @@ export class MeshBuilder {
     public static CreateDecal(name: string, sourceMesh: AbstractMesh, options: { position?: Vector3, normal?: Vector3, size?: Vector3, angle?: number }): Mesh {
         return DecalBuilder.CreateDecal(name, sourceMesh, options);
     }
+
+
+    /**
+     * Creates a decal mesh.

```

Comment:
wrong doc ;)

Reply:
Ill fix all that asap!  I had no time to do docs yet.